exonum/src/messages/mod.rs

```diff
@@ -13,255 +13,209 @@
 // limitations under the License.
 
 //! Consensus and other messages and related utilities.
+use std::fmt;
```

Comment:
I would like to see the state machine described in module level documentation here.